sonic-xcvrd/xcvrd/xcvrd_utilities/y_cable_helper.py

```diff
@@ -42,6 +43,25 @@
 MUX_CABLE_STATIC_INFO_TABLE = "MUX_CABLE_STATIC_INFO"
 MUX_CABLE_INFO_TABLE = "MUX_CABLE_INFO"
 
+def format_the_string(string):
+    """
+    Takes an arbitrary string and creates a valid entity for port mapping file.
+    The input could contain trailing and leading spaces, upper cases etc.
+    Convert them to what is defined in the y_cable vendor_mapping file.
+
+    """
+
+    # create a working copy (and make it lowercase, while we're at it)
+    s = string.lower()
+
+    # remove leading and trailing whitespace
+    s = s.strip()
+
+    # Make spaces into underscores
+    s = re.sub('[\\s\\t\\n]+', '_', s)
```

Comment:
Do you need `\t` and `\n`? I believe `\s` should cover all whitespace.

Also suggest using a raw string for the pattern, which eliminates the need for escaping the backslashes. E.g., `s = re.sub(r'\s+', '_', s)`

Also suggest changing comment to `# Replace whitespace with underscores`

Reply:
fixed